packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -499,6 +499,7 @@ export function renderTable<T extends ScalarDict>({renderOptions, ...props}: Ren
 
 interface RenderTasksOptions {
   renderOptions?: RenderOptions
```

Comment:
I explored overriding `stdout` in renderOptions (this comes from Ink), but I felt that it felt really hacky and required users to still pipe their output

If there's a way to point a `tty.writestream` to `/dev/null` I'd be open to exploring that as well!